### PR TITLE
Tidy up the frontend app

### PR DESCRIPTION
**Pull request by valo**
DRY up the views using layouts and components.

### CODE REVIEW

File: app/components/projects-table/index.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+                            { txOut.toLocaleString('en-US') }
+                          </div>
+                        )
+                      })
+                    }
+                    </td>
+                  </tr>
+                )
+              })
+            }
```
Comment (valo) [inline app/components/projects-table/index.js]: Can we split these maps into separate functions, which are named properly and this way make the whole template a bit shorter? I think is going to make this template a bit more readable. What do you think?

File: app/components/topbar/index.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  <div className="row topbar">
+    <div className="col-lg-6">
+      <div style={{'paddingTop': '24px', 'paddingLeft': '16px'}}>
+        <i className="material-icons">search</i>
+      </div>
+    </div>
+    <div className="col-lg-6">
+      <ul className="nav-right pull-right list-unstyled">
+        <li>
+          <span style={{'display': 'inline-block', 'paddingTop': '22px', 'paddingLeft': '24px', 'paddingRight': '24px', 'fontSize': '14px'}}>12.5 Ξ</span>
```
Comment (valo) [inline app/components/topbar/index.js]: Can we use styled jsx for the CSS? This is the standard way to handle CSS in the next.js bundle: https://github.com/zeit/styled-jsx It is even possible to split the CSS into a separate file and import it in the JS file and then use `styleName` to refer to a class. The nice thing is that the CSS is being scoped to the current component and won't impact any HTML elements outside of it.

File: app/pages/cashflow.js [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+        wallets: [
+          {
+            last_outgoing: null,
+            balance: null,
+            tx_out: null
+          }
+        ]
+      }
+    ],
+    eth_price: 2
```
Comment (valo) [inline app/pages/cashflow.js]: Can we keep the API call here? We will need to use the API for the code we merge into master

File: app/components/projects-table/index.js (hunk 1 above)
Comment (bobrovova) [reply app/components/projects-table/index.js]: I agree with you. Done

File: app/components/topbar/index.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
+import styles from './index.scss'
+
+export default (props) => (
+  <div className="row topbar">
+    <style dangerouslySetInnerHTML={{ __html: styles }}></style>
+    <div className="col-lg-6">
+      <div className="search" style={{'paddingTop': '24px', 'paddingLeft': '16px'}}>
```
Comment (valo) [inline app/components/topbar/index.js]: Can we move this in-like styles to the css file of the component?

File: app/pages/roadmap.js [hunk 5]
```diff
@@ -1,287 +1,214 @@
...
-        <div class="row topbar">
-            <div class="col-lg-6">
-                <div style="padding-top: 24px; padding-left: 16px;">
-                    <i class="material-icons">search</i>
+    <MainHead />
+    <SideMenu activeItem="roadmap"/>
+    <div className="container vert-stretch" id="main">
+      <Topbar />
+      <HeaderPage name="SANbase Roadmap" description="Please see our SANbase roadmap below. It is a living document; milestones may adjust." />
+      <style dangerouslySetInnerHTML={{ __html: styles }}></style>
```
Comment (valo) [inline app/pages/roadmap.js]: Why do we need to inject the css like this? Isn’t it possible to just use the classes in the ‘style’ object?

File: app/components/header-page/index.js [hunk 6]
```diff
@@ -0,0 +1,12 @@
+import styles from './roadmap.scss'
```
Comment (valo) [inline app/components/header-page/index.js]: This import can't be working as this file is located into another folder

File: app/pages/cashflow.js [hunk 7]
```diff
@@ -1,197 +1,98 @@
...
-                }
-            }
+      });
+    });
+  }
 
-        });
+  static async getInitialProps(){
+    //const res = await fetch(WEBSITE_URL + '/api/cashflow')
+    //const data = await res.json()
```
Comment (valo) [inline app/pages/cashflow.js]: We should be fetching the data from the server, so please enable this.

**Pull request merged**